fail2ban/server/server.py

```diff
@@ -324,6 +324,15 @@ def setUnbanIP(self, name, value):
 	def getBanTime(self, name):
 		return self.__jails[name].actions.getBanTime()
 	
+	def isAlive(self, jailnum=None):
+		if jailnum is not None and len(self.__jails) != jailnum:
+			return 0
+		for j in self.__jails:
+			j = self.__jails[j]
```

Comment:
On Fri, 24 Jul 2015, Serg G. Brester wrote:
>  > +     def isAlive(self, jailnum=None):
>  > +             if jailnum is not None and len(self.__jails) != jailnum:
>  > +                     return 0
>  > +             for j in self.__jails:
>  > +                     j = self.__jails[j]

>    no, it's correct so, and without it it's broken (__jails is a "dict")...

ah -- right!  See -- that is where naming conventions lead to
confusion(s). If it was 

for jail_name in self.__jails:
   jail = self.__jails[jail_name]

I could easily understand what is going on despite the late time in the
evening.  But altogether then it should have been

for jail in self.__jails.itervalues():
    if not jail.isAlive():
        return False

correct? (note me changing 0,1 to more appropriate bool)

>    Why should I prove for you the accuracy of each line? A little trust must
>    be also, or not?

If I didn't trust you I would have not given push rights ;)  It is a
matter of making myself familiar with the changes and seeing if there is
no hidden issues and either code could be easily maintained in the long run.